<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cpp-Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>441</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Message:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_subject">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>391</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Subject:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_subject">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>411</y>
      <width>421</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_to">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>341</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_to">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>361</y>
      <width>421</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>460</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="Line" name="separator">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>621</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chat:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="separator_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>16</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>141</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chats:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_to</tabstop>
  <tabstop>lineEdit_subject</tabstop>
  <tabstop>plainTextEdit_message</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
